ccd-definition/ComplexTypes/DocumentReference.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Does this not do the same thing as the Document field type?

Reply:
it does, removed :+1: